content/en/docs/languages/net/instrumentation.md

```diff
@@ -63,320 +279,362 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .AddConsoleExporter()
     .Build();
 
-// ...
-```
-
-This is also where you can configure instrumentation libraries.
+var meterProvider = Sdk.CreateMeterProviderBuilder()
```

Comment:
since we are removing "using" keyword from here, we need to explicitly dispose() at the end of the application, else metrics won't be flushed to console! (Similar concern for logs/traces too, though logs/traces will still show up in console even without dispose, but we should show the right thing that works across all exporters.)
in short - please add explicit .Dispose() call on meterProvider, tracerProvider, loggerFactory at the end of the application.